inference-engine/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/embedding_bag_packed_sum.cpp

```diff
@@ -12,20 +12,20 @@ using namespace LayerTestsDefinitions;
 
 namespace {
 
-const std::vector<InferenceEngine::Precision> netPrecisions = {
-    InferenceEngine::Precision::FP32,
-    InferenceEngine::Precision::FP16,
+const std::vector<ov::test::ElementType> netPrecisions = {
+    ov::test::ElementType::f32,
+    ov::test::ElementType::f16,
 };
 
-const std::vector<InferenceEngine::Precision> indPrecisions = {
-    InferenceEngine::Precision::I64,
-    InferenceEngine::Precision::I32
+const std::vector<ov::test::ElementType> indPrecisions = {
+    ov::test::ElementType::i64,
+    ov::test::ElementType::i32
 };
 
-const std::vector<std::vector<size_t>> emb_table_shape = {
-    {5, 6},
-    {10, 35},
-    {5, 4, 16}
+const std::vector<ov::test::InputShape> input_shapes = {
+    {{5, 6}, {{5, 6}}},
```

Comment:
as far as I understood here should be
```suggestion
    {{}, {{5, 6}}},
```

Reply:
discussed offline, static shape case: {5, 6} input model shape and {5, 6} - input data shape